Split test results 2 into multiple files

tonyfast opened the pull request:
- [x]  split results.md
- [x] add to the documentation toc

cc: @Jenneh

tonyfast commented:
@Jenneh you can preivew the layout here: https://iota-school.github.io/notebooks-for-all/branch/results-split/exports/user-tests/2-content/results/

Jenneh commented:
Hello, This looks great! I found one issue that needs to be resolved before we make this live. Currently, clicking "conclusions" leads to a 404 page.

tonyfast commented on mkdocs.yml:
```diff
@@ -45,7 +45,15 @@ nav:
+        - Cell contents: exports/user-tests/2-content/results/cell-contents.md
+        - Rich outputs: exports/user-tests/2-content/results/rich-outputs.md
+        - Navigation: exports/user-tests/2-content/results/navigation.md
+        - Conclusions: exports/user-tests/2-content/results/conclusions.md
```
```suggestion
        - Conclusions: exports/user-tests/2-content/results/conclusion.md
```

tonyfast commented on user-tests/2-content/results/conclusion.md:
```diff
@@ -0,0 +1,25 @@
+# Issues
+
+Bugs, issues, and other specific feedback or discussions from this round of tests can be found throughout the repository in issues. Listed below, they are
```
```suggestion
Bugs, issues, and other specific feedback or discussions from this round of tests can be found throughout the repository in issues. Listed below, they are

```

tonyfast replied:
add a new line to invoke the list rendering

tonyfast commented:
thanks for testing this @Jenneh i fixed the table of contents to avoid a 404 and cleaned up a list. im going to merge this so it hits the main and site and we can defer other content changes to later.
